himportscan/src/HImportScan/GHC.hs

```diff
@@ -1,5 +1,38 @@
+{-# LANGUAGE CPP #-}
```

Comment:
Rather than using CPP here, I would prefer to have a version of this file for each supported compiler.

Reply:
I had a similar idea as well, I can update the PR. How do you want to decide what file to import in the other module, by using cpp just for the import?

Reply:
Ideally, bazel would pickup the right file to compile and link based on the ghc version. Only the build system would be involved and not the source file contents. But I don't have a good grasp of how easy this is currently.